### PR TITLE
Add first version of Zod locales

### PR DESCRIPTION
**Pull request by lukasvice**

### CODE REVIEW

File: src/react/ZodInternationalization.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { useZodInternationalization } from './useZodInternationalization'
+
+export function ZodInternationalization() {
```
Comment (mmalfertheiner) [inline src/react/ZodInternationalization.ts]: I would be more explicit with the naming and call it `ZodInternationalizationWithReactIntl`

**Review by mmalfertheiner — CHANGES_REQUESTED** [review]

File: src/react/ZodInternationalization.ts (hunk 1 above)
Comment (lukasvice) [reply src/react/ZodInternationalization.ts]: Ok, that's a long word :)

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+npm install @aboutbits/zod-locales-formatjs
+```
+
+Second, add the custom language packages to your `IntlProvider`:
+
+```js
+import { de } from '@aboutbits/zod-locales-formatjs'
+import { IntlProvider } from 'react-intl'
+
+<IntlProvider messages={{...youMessages, ...de }} {...}>{children}</IntlProvider>
```
Comment (mmalfertheiner) [inline readme.md]:
```suggestion
<IntlProvider messages={{...yourMessages, ...de }} {...}>{children}</IntlProvider>
```

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export * from './zod/makeZodErrorMap'
+export * from './zod/errorMessages'
+
+export { default as de } from './translations/de.json'
+export { default as it } from './translations/it.json'
+
+export * from './react/useZodInternationalizationWithReactIntl'
+export * from './react/ZodInternationalizationWithReactIntl'
```
Comment (mmalfertheiner) [inline src/index.ts]: I think this will force you to install `react-intl`, otherwise it will not compile.
Maybe we could export it from `react-intl.ts`?

Comment (lukasvice) [reply src/index.ts]: Good point. I'll try it

**Review by mmalfertheiner — APPROVED** [review]

**Pull request merged**